src/Dfc.CourseDirectory.Core/DataStore/Sql/Queries/DeleteLarsLearningDelivery.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using OneOf;
+using OneOf.Types;
+
+namespace Dfc.CourseDirectory.Core.DataStore.Sql.Queries
+{
+    public class DeleteLarsLearningDelivery : ISqlQuery<OneOf<string, None>>
+    {
+        public string LearnAimRef { get; }
```

Comment:
We tend to use POCOs here instead of read-only properties and a non-default constructor (bring on C# 9 records)